src/components/introduction/ChangeLog/changeLog.stories.tsx

```diff
@@ -15,6 +15,7 @@ import v0_4_3 from './logs/0.4.3.json'
 import v0_4_4 from './logs/0.4.4.json'
 import v0_4_6 from './logs/0.4.6.json'
 import v0_5_0 from './logs/0.5.0.json'
+import v0_6_0 from './logs/0.6.0.json'
```

Comment:
The Changelog is usually updated in the develop, after the merge and before you publish a new version of the package ^^

Reply:
Yes make sense will do it this way next time 👍